src/lib/annotations/Annotator.js

```diff
@@ -504,16 +564,15 @@ class Annotator extends EventEmitter {
             return;
         }
 
+        // Exits point annotation mode on first click
+        this.togglePointModeHandler();
```

Comment:
The way it was before, togglePointModeHandler was called on an invalid location off the document as well.